src/test/resources/datasets/users_dataset.xml.hbs

```diff
@@ -10,7 +10,7 @@
 
     <user_role user_id="{{user_id}}" role="Privilege Level: Full" />
 
-    <user_property user_id="{{user_id}}" property="defaultLocale" property_value="ht" />
+    <user_property user_id="{{user_id}}" property="defaultLocale" property_value="{{defaultLocale}}" />
```

Comment:
I'm probably missing it, but I assume this gets set somewhere earlier?

Reply:
I have no idea. I deleted `ht` and typed `{{` and then `default_locale` showed up as an autocomplete option, and I said "well that sounds right," and then it worked correctly. I don't know where these variables come from.

Reply:
I suspect that is just setting the "defaultLocale" property of the uesr to literally "{{defaultLocale}}" and will break some of other tests.   The use case here I'm guessing is that we want the default locale for the user to be Spanish for the Peru and Mexico smoke tests?

Reply:
Oh, no way. It's crazy to me that the tests and everything worked just fine, displaying in Spanish, with the default locale property set like that.

Reply:
Yeah, not sure, but a bunch of the HUM-CI tests are failing now and it looks like the errors are from not finding Kreyol text they are searching for... @brandones